addons/mail/static/tests/thread/thread_tests.js

```diff
@@ -213,6 +213,34 @@ QUnit.test("thread is still scrolling after scrolling up then to bottom", async
     await contains(".o-mail-Thread", { scroll: "bottom" });
 });
 
+QUnit.test("mention a channel with group restriction", async () => {
```

Comment:
I would slightly reword name of test to help understand the purpose of test much quicker: `Can mention other channels in a group-restricted channel`